readme.md

```diff
@@ -88,25 +87,24 @@ final case class Config(
 
 object Main extends IOApp {
   def run(args: List[String]): IO[ExitCode] =
-    Blocker[IO].use { blocker =>
-      val config =
-        configMapInNamespace("pizza", blocker).flatMap { configMap =>
-          (
-            configMap("pizzaBrand"), // Key can be omitted if config map has only one entry
-            configMap("delivery", "radius").as[Int], // Key is necessary if config map has multiple entries
-            configMap("delivery", "charge").as[Boolean]
-          ).parMapN { (pizzaBrand, deliveryRadius, isDeliveryCharge) =>
-            Config(
-              appName = "my-pizza-api",
-              pizzaBrand = pizzaBrand,
-              deliveryRadius = deliveryRadius,
-              isDeliveryCharge = isDeliveryCharge
-            )
-          }
+    val config =
+      configMapInNamespace[IO]("pizza", blocker).flatMap { configMap =>
```

Comment:
I might have missed something, but where's the blocker coming from now?

Reply:
Blocker got removed from Cats-effect 3. I missed that entry on the readme. will fix it

Reply:
I should probably add mdoc so the readme would be compiled :)